runfile.js

```diff
@@ -11,5 +11,5 @@ export function echo(){
 }
 
 export function build(){
-    run('babel src/index.js -o lib/index.js');
+    run('babel src/ --out-dir lib/');
```

Comment:
It's just a little future proofing. Let's you break down the index in the future and everything still work.